constraint/deploy/crds.yaml

```diff
@@ -17,14 +17,10 @@ spec:
     openAPIV3Schema:
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
For some reason you are generating non-wrapped manifests?

Reply:
This repo has controller-gen v0.1.8, that's probably the reason we are seeing non-wrapped lines

Reply:
looks like this is due to kustomize version, repo uses v3.0.2 and i have v3.7.0 locally

Reply:
Interesting, I wonder why Kustomize version affects this?

Reply:
NVM, I was thinking kubebuilder, not kustomize

Reply:
@brycecr @juliankatz

It looks like version skew of kustomize is what's causing the different wrappings for the manifest. Do we want to Dockerize kustomize as well?